lib/utils/SchemaBuilderXSD.js

```diff
@@ -407,7 +414,13 @@ class SchemaBuilderXSD {
     else {
       localSchemaTag[parserPlaceholder + 'xmlns:' + schemaNamespace.key] = schemaNamespace.url;
     }
-    localSchemaTag[parserPlaceholder + 'xmlns:tns'] = tnsNamespaceURL;
+    if (tnsNamespace) {
```

Comment:
is there any scenario where tnsNamespace is not set? If we are always creating the tnsNamespace object (either if the prefix is tns or not, should be present right?) so maybe we should remove the else statement.

Reply:
@LuisTejedaS `tnsNamespace` could be empty in case when there is no `tns` namespace is mentioned. (No namespace with same URL namespace as `targetNamespace`)

Reply:
got it, thanks